packages/cli/src/commands/index.ts

```diff
@@ -22,6 +22,11 @@ export function registerCommands(program: CommanderStatic) {
     .command('app:build')
     .description('Build an app for a production release')
     .option('--stats', 'Write bundle stats to output directory')
+    .option(
+      '--parallel [value]',
```

Comment:
Would you be alright with only providing the env var for now? It's still a bit of a niche use-case and we do wanna try to keep the CLI lean.